Bottom: b8386f53d00f8b85c6231e904e3724a50703b000
Top:    3ac6ab007adb37525f56b8e4f0f8b21b19fee3e0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-04 01:23:51 +0000

Refresh of tag-circular-contours-so-they-

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index a582d3d..76953d1 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -602,7 +602,6 @@ do_hole (const BoxType *b, void *cl)
 {
   struct do_hole_info *info = cl;
   PLINE *curc = (PLINE *) b;
-  cairo_traps_t *traps;
 
   /* Ignore the outer contour - we draw it first explicitly*/
   if (curc->Flags.orient == PLF_DIR) {
